studiopress/simple-social-icons
# Add TripAdvisor icon

**Summary of change:**

Add TripAdvisor icon

**Have the changes in this PR been added to the documentation for this project?**

Yes

**How to test:**

Look for TripAdvisor URI field in widget
Add a URI to the TripAdvisor URI field
Check that the TripAdvisor icon has appeared
